input/pagecontent/reference.md

```diff
@@ -682,6 +830,7 @@ The following table is a summary of the rules applicable to profiles, extensions
 | Insert | <code>* insert {RuleSet name}<i>({value1}, {value2}, ...)</i></code> |
 | Obeys | `* obeys {Invariant id}` <br/> `* obeys {Invariant1 id} and {Invariant2 id} ...` <br/> `* <element> obeys {Invariant id}` <br/> `* <element> obeys {Invariant1 id} and {Invariant2 id} ...` |
 | Type | `* <element> only {datatype}` <br/> `* <element> only {datatype1} or {datatype2} or {datatype3} ...` <br/> `* <element> only Reference({ResourceType name|id|url})` <br/> `* <element> only Reference({ResourceType1 name|id|url} or {ResourceType2 name|id|url} or {ResourceType3 name|id|url} ...)`|
+| Path | `* <element>`|
```

Comment:
A path rule is also allowed on Mappings to apply context, but I don't see a place where there is an equivalent table for rules on Mappings. Does it make sense to highlight path rules as a part of Mappings? Or is that not necessary for this documentation?

Reply:
Added a note here: 64c2548.